### PR TITLE
Created issue templates

### PR DESCRIPTION
**Pull request by Aarjav-Jain**
Issue #1 
Created issue templates using https://github.com/appwrite/appwrite/tree/master/.github/ISSUE_TEMPLATE as a reference

### CODE REVIEW

**Comment by Aarjav-Jain** [comment]
Please review and check if it requires any changes

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:40]:
```suggestion
      label: PHP Version
```

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - type: textarea
+    id: steps-to-reproduce
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: dropdown
+    id: version
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:38]: please also add Utopia Preloader Version option.

File: .github/ISSUE_TEMPLATE/feature.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    attributes:
+      label: "🔖 Feature description"
+      description: "A clear and concise description of what the feature is."
+      placeholder: "You should add ..."
+  - type: textarea
+    id: pitch
+    validations:
+      required: true
+    attributes:
+      label: "🎤 Pitch"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/feature.yaml:23]: Add code of conduct in this form too.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Some trivial changes, please take this as reference
https://github.com/utopia-php/framework/pull/28

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 2 above)
Comment (Aarjav-Jain) [reply .github/ISSUE_TEMPLATE/bug.yaml:38]: what will be the option in for Utopia Preloader Version dropdown ?

**Comment by lohanidamodar** [comment]
@Aarjav-Jain please still more changes required to make it consistent with
https://github.com/utopia-php/framework/pull/28.

Like format of code of conduct field, remove extra fields. Please make the update accordingly.
Thank you.

**Comment by Aarjav-Jain** [comment]
Should i remove the contact details, what happened and relevant log output labels?
Do you want it exactly same as in [https://github.com/thinkverse/issues-test/blob/main/.github/ISSUE_TEMPLATE/bug.yaml]

**Comment by lohanidamodar** [comment]
@Aarjav-Jain Yes please, we want it consistent across our repositories at the moment.

**Comment by Aarjav-Jain** [comment]
Done Pls review!

**Comment by Aarjav-Jain** [comment]
@lohanidamodar pls review

**Comment by christyjacob4** [comment]
@Aarjav-Jain closing this PR for now since there has been no activity in quite a long time. Feel free to reopen it when you're ready to start working on it once again. 🙂

**Pull request closed without merging**